patches/third_party-widevine-cdm-widevine.gni.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third_party/widevine/cdm/widevine.gni b/third_party/widevine/cdm/widevine.gni
+index 82a93622585a424f56c3567050f9ba4822de6c1e..9c89d9a0c0a8b2df4e082cfd1ef9ace6ed8fc929 100644
+--- a/third_party/widevine/cdm/widevine.gni
++++ b/third_party/widevine/cdm/widevine.gni
```

Comment:
we don't need third_party/widevine because we don't bundle widevine cdm.

Reply:
nvm, just saw `enable_widevine_cdm_host_verification` also used in chrome/BUILD.gn for generating widevine resource bundle and signing framework instead of just used only with `should_bundle_widevine_cdm`

Reply:
please rebase your patch to 70.0.3538.22 so we can get rid of this patch
```
 13 enable_widevine_cdm_host_verification =
 14     enable_widevine && enable_cdm_host_verification
 15
 16 # Only bundle Widevine CDM in Google Chrome builds.
 17 should_bundle_widevine_cdm = is_chrome_branded && enable_library_cdms

```

Reply:
it turns out I have previously local change applied and this patch will be removed in the future. https://chromium.googlesource.com/chromium/src/+/f170c737755811ca5d7ded0700d8aa17ec310024